assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -161,54 +159,52 @@ export default function SettingsView() {
 					</p>
 				</div>
 			</div>
-			{ isGA4Enabled &&
-				ga4PropertyID &&
-				ga4PropertyID !== PROPERTY_CREATE && (
-					<div className="googlesitekit-settings-module__meta-items">
-						<div className="googlesitekit-settings-module__meta-item">
-							<h5 className="googlesitekit-settings-module__meta-item-type">
-								{ __(
-									'Google Analytics 4 Property',
+			{ ga4PropertyID && ga4PropertyID !== PROPERTY_CREATE && (
+				<div className="googlesitekit-settings-module__meta-items">
+					<div className="googlesitekit-settings-module__meta-item">
+						<h5 className="googlesitekit-settings-module__meta-item-type">
+							{ __(
+								'Google Analytics 4 Property',
+								'google-site-kit'
+							) }
+						</h5>
+						<p className="googlesitekit-settings-module__meta-item-data">
+							<DisplaySetting value={ ga4PropertyID } />
+						</p>
+					</div>
+					<div className="googlesitekit-settings-module__meta-item">
+						<h5 className="googlesitekit-settings-module__meta-item-type">
+							{ createInterpolateElement(
+								__(
+									'<VisuallyHidden>Google Analytics 4</VisuallyHidden> Measurement ID',
 									'google-site-kit'
-								) }
-							</h5>
-							<p className="googlesitekit-settings-module__meta-item-data">
-								<DisplaySetting value={ ga4PropertyID } />
-							</p>
-						</div>
-						<div className="googlesitekit-settings-module__meta-item">
-							<h5 className="googlesitekit-settings-module__meta-item-type">
+								),
+								{
+									VisuallyHidden: <VisuallyHidden />,
+								}
+							) }
+						</h5>
+						<p className="googlesitekit-settings-module__meta-item-data">
+							<DisplaySetting value={ ga4MeasurementID } />{ ' ' }
+							<Link
+								href={ editDataStreamSettingsURL }
+								external
+								inherit
+							>
 								{ createInterpolateElement(
 									__(
-										'<VisuallyHidden>Google Analytics 4</VisuallyHidden> Measurement ID',
+										'Edit <VisuallyHidden>Google Analytics 4 web data stream </VisuallyHidden>in Analytics',
 										'google-site-kit'
 									),
 									{
 										VisuallyHidden: <VisuallyHidden />,
 									}
 								) }
-							</h5>
-							<p className="googlesitekit-settings-module__meta-item-data">
-								<DisplaySetting value={ ga4MeasurementID } />{ ' ' }
-								<Link
-									href={ editDataStreamSettingsURL }
-									external
-									inherit
-								>
-									{ createInterpolateElement(
-										__(
-											'Edit <VisuallyHidden>Google Analytics 4 web data stream </VisuallyHidden>in Analytics',
-											'google-site-kit'
-										),
-										{
-											VisuallyHidden: <VisuallyHidden />,
-										}
-									) }
-								</Link>
-							</p>
-						</div>
+							</Link>
+						</p>
```

Comment:
This diff is totally wrong. I only removed `isGA4Enabled`. This is definitely one of the few cases where Prettier is a disadvantage :thinking: 

I almost want to replace `{ isGA4Enabled &&` with `{ true &&` just to keep a clear diff :see_no_evil:  (or alternatively disable prettier here, which could be removed on a future PR :thinking: )